### ISSUE TITLE
ilastik plugin for fiji

### ISSUE THREAD

**Issue by lldelisle** [ilastik/ilastik-build-conda#111]
Dear ilastik conda recipe developers,
I would like to have ilastik working with fiji on a conda environment. Fiji is already available in conda (bioconda channel: https://anaconda.org/bioconda/fiji). I saw on your documentation that the way to install the ilastik plugin is to use the plugin manager from Fiji but I was wondering if it would be possible to install it just by using the jar and just writting a simple recipe like https://github.com/bioconda/bioconda-recipes/tree/master/recipes/fiji-morpholibj
Thanks for your feedback.
PS: If you agree on the principle and tell me where I can find the jar, I can write the recipe and test.

**Comment by lldelisle** [comment]
Update:
I found the `jar` in https://maven.scijava.org/service/local/repositories/releases/content/org/ilastik/ilastik4ij/1.8.2/ilastik4ij-1.8.2.jar.
Just copying the jar in `${CONDA_PREFIX}/share/plugins/` works.
To set the path of the `./run_ilastik.sh`. I seems that:
```
echo "run(\"Configure ilastik executable location\", \"executablefile=/path/to/run_ilastik.sh\");" > exec.ijm
ImageJ --ij2 --headless --console -macro exec.ijm
```
Works so I am close to write a PR. I still have an issue with installing ilastik in conda but I will open another issue for that.

**Comment by k-dominik** [comment]
Hello @lldelisle,

sorry for the late reply. Yes, copying the jar is usually enough. Our repo for  the fiji plugin is here, btw: https://github.com/ilastik/ilastik4ij



Happy to help with the ilastik conda-installation (since recently it shoud be a little easier https://github.com/ilastik/ilastik#conda-installation-experimental )

**Comment by lldelisle** [comment]
Thanks, it helped indeed. My PR is available at #112
